### ISSUE TITLE
S3: uploading to dest with slash in beginning should fail but doesn't

### ISSUE THREAD

**Issue by davidduffett** [jpillora/grunt-aws#21]
When you specify a "dest" option that begins with a slash, eg:

    {
      src: "**",
      dest: "/myfolder/"
    }

The task output displays like it has succeeded, eg:

Running "s3:assets" (s3) task
Retrieving list of existing objects prefixed with '/myfolder/'...
Put '/myfolder/myfile.html'

When actually nothing is uploaded at all.

The task should probably fail in these cases, as a slash is not valid at the beginning of an object identifier in S3.

**Comment by jpillora** [comment]
Yep, thats a bug, not sure when I'll get time to fix it though...
